hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterClientProtocol.java

```diff
@@ -919,10 +919,10 @@ public HdfsFileStatus getFileInfo(String src) throws IOException {
         if (dates != null && dates.containsKey(src)) {
           date = dates.get(src);
         }
-        ret = getMountPointStatus(src, children.size(), date);
+        ret = getMountPointStatus(src, children.size(), date, false);
```

Comment:
It looks like false is kind of the exception for this method.
Can we keep the old signature with `true` as the default?